src/pages/Address/index.tsx

```diff
@@ -111,94 +110,131 @@ enum PageParams {
   MaxPageSize = 100,
 }
 
+interface TransactionData {
+  transactionWrappers: TransactionWrapper[]
+  totalTransactions: number
+  pageNo: number
+  pageSize: number
+}
+
+const initAddress: Address = {
+  address_hash: '',
```

Comment:
why use another naming convention?

Reply:
These fields come from http response which is defined by backend.

Reply:
please open a new issue